nyxx_lavalink/lib/src/node/nodeRunner.dart

```diff
@@ -61,7 +61,7 @@ Future<void> _handleNode(SendPort clusterPort) async {
             process(jsonDecode(data as String) as Map<String, dynamic>);
           }, onDone: () async {
             clusterPort.send({"cmd": "DISCONNECTED", "nodeId": node.nodeId});
-            connect();
+            await connect();
```

Comment:
why you added await there?

Reply:
[Done](https://github.com/nyxx-discord/nyxx/pull/171/commits/1e1f33532db3088a09b5cb0e957cd15b1249a8ea)

Reply:
The analyzer complained that `connect()` was not awaited